miner/worker.go

```diff
@@ -1080,6 +1081,32 @@ func (w *worker) fillTransactions(interruptCh chan int32, env *environment, stop
 			localBlobTxs[account] = txs
 		}
 	}
+
+	if bidTxs != nil {
+		var wg sync.WaitGroup
+		wg.Add(4)
+
+		filterBidTxs := func(commonTxs map[common.Address][]*txpool.LazyTransaction) {
+			defer wg.Done()
+			for acc, txs := range commonTxs {
+				newTxs := make([]*txpool.LazyTransaction, 0)
+				for _, tx := range txs {
+					if !bidTxs.Contains(tx.Hash) {
+						newTxs = append(newTxs, tx)
+					}
+				}
+				commonTxs[acc] = newTxs
+			}
+		}
+
+		go filterBidTxs(localPlainTxs)
```

Comment:
I dont think there is a need to start 4 Go routine:
1. Blob Txs is quite few on the network;
2. local Txs is quite few or none for a validator;

so only `filterBidTxs(remotePlainTxs)` will cost most of the time.

Reply:
ok

Reply:
emm, the current implementation get too many duplicated code, pease do optimization.

Reply:
fixed